packages/styling/etc/styling.api.md

```diff
@@ -448,6 +448,17 @@ export type ISchemeNames = 'default' | 'neutral' | 'soft' | 'strong';
 // @public
 export interface ISemanticColors extends ISemanticTextColors {
     accentButtonBackground: string;

```

Comment:
we should also figure out where we're headed with the "accent" color, very very few designs make use of it (intentionally perhaps?)